mesh/mesh.py

```diff
@@ -56,30 +59,48 @@ def mesh_to_buffers(mesh: bpy.types.Mesh) -> MeshBuffers:
   positions = tmp_vec3[indices]
 
   # read normals (these contain pre-calculated normals handling  flat, smooth, custom split normals)
-  corner_norm = np.empty((len(mesh.corner_normals), 3), dtype=np.float32)
-  mesh.corner_normals.foreach_get('vector', corner_norm.ravel())
+  if bpy.app.version < (4, 0, 0):
+      mesh.calc_normals_split()
+      corner_norm = np.empty((len(mesh.loops) * 3, 3), dtype=np.float32)
+      mesh.loops.foreach_get('normal', corner_norm)
```

Comment:
I didn't test but I'm surprised this isn't
```suggestion
      corner_norm = np.empty((len(mesh.loops), 3), dtype=np.float32)
      mesh.loops.foreach_get('normal', corner_norm.ravel())
```
(don't just blindly apply that, I'm just wondering)